src/Main.java

```diff
@@ -0,0 +1,96 @@
+
+import com.sun.source.tree.IfTree;
+
+import java.io.IOException;
+import java.util.*;
+
+public class Main {
+
+    static final String INPUT_FILEPATH = "testfiles/testIn.csv";
+    static final String OUTPUT_FILEPATH = "outputFile.csv";
+
+    public static void main(String[] args) {
+
+        //Read in the .csv file
+        List<Map<String, String>> rowsOfData = CsvReader.readCsv(INPUT_FILEPATH);
+
+        //find the lowest origin year and range of years, for the first line of the output.
+        int lowestOriginYear = 0;
```

Comment:
Id separate this block, lines 18-55 into its own class called ProductMapper which would convert a List<Row> into a List<Product>

Its better to have lots of appropriately named small classes than a few larger classes. Each class is meant to do one thing, this makes it easier to find where we need to make changes when we get new requirements and makes it less likely for us to produce bugs when we do make changes as things are more isolated.

It also helps with unit testing as you test code class by class. This makes it easier to cover every line with tests and if someone makes a breaking change it makes it more likely that it will be caught by the unit test